docs/docs/guides/compatibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: Compatibility
+title: Compatibility
+sidebar_label: Compatibility
+---
+
+# Compatibility Table
+
+In order to provide solid, state-of-the arts solutions we often use brand new React Native features,
+that are being introduced all the time. Thats why applications with mismatched Reanimated and React Native
+versions may show unexpected behavior or even fail to build at all. The table below contains official list
+of supported React Native versions.
+
+#### Supported React Native versions - New Architecture (Fabric) enabled
+
+| Reanimated version(s)       | The oldest supported React Native | The newest supported React Native |
+| --------------------------- | --------------------------------- | --------------------------------- |
+| from **3.1.x** to **3.5.x** | 0.72                              | 0.72                              |
+
+#### Supported React Native versions - New Architecture NOT enabled
```

Comment:
```suggestion
#### Supported React Native versions

##### New Architecture (Fabric)

| Reanimated version(s)       | The oldest supported React Native | The newest supported React Native |
| --------------------------- | --------------------------------- | --------------------------------- |
| from **3.1.x** to **3.5.x** | 0.72                              | 0.72                              |

##### Legacy architecture (Paper)
```

Reply:
I've tried searching in google both `old architecture react native` and `legacy architecture react native` and found no articles using terminology "legacy architecture". It seems that the actual terminology for paper is either "old architecture" or "legacy native components"